frontend/src/components/header/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { IonHeader, IonPage, IonTitle, IonToolbar } from '@ionic/react'
+import { PropsWithChildren } from 'react'
+
+interface HeaderProps {
+  pageTitle: string
+}
+
+const Header: React.FC<HeaderProps> = ({ pageTitle }) => {
```

Comment:
I would recommend to stick with ```Props``` and only use named interfaces when exporting it.

```suggestion
interface Props {
  pageTitle: string
}

const Header: React.FC<Props> = ({ pageTitle }) => {
```